Bottom: 35f4c9c7575599a63f2ef81b09754b2704c731bf
Top:    bccfdad6491fa081f2d2f895fd829434d3894273
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-25 01:40:31 +0000

Refresh of revert-find-c-remove-undo-para

---

diff --git a/src/select.c b/src/select.c
index 27a590b..0461778 100644
--- a/src/select.c
+++ b/src/select.c
@@ -1029,7 +1029,8 @@ SelectObjectByName (int Type, char *Pattern, bool Flag)
           {
             for (i = menu->EntryN, entry = menu->Entry; i; i--, entry++)
               if (SeekPad (entry, &conn, false))
-                RatFindHook (conn.type, conn.ptr1, conn.ptr2, conn.ptr2, true);
+                RatFindHook (conn.type, conn.ptr1, conn.ptr2, conn.ptr2,  
+                             true, true);
           }
       }
       END_LOOP;
